docs/swag.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'wasmCloud Swag'
+sidebar_position: 20
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import HubspotForm from 'react-hubspot-form';
+
+
+# wasmCloud Swag
```

Comment:
I think we should write out a bit more on this page, what it's for, information folks need to provide, etc

Reply:
Should this be open only to folks that have at least one good-first-issue closed? US Residents only?

Reply:
There is some extra information on the HubSpot form itself. It's only open to US folks, but we could include a comment to say not all submissions are guaranteed a swag-kit and contributors are prioritized so be sure to include your github id?